tools/functional-tester/etcd-tester/tester.go

```diff
@@ -25,6 +25,11 @@ type tester struct {
 	limit           int
 	status          Status
 	currentRevision int64
+
+	Stressers        []Stresser
+	stressBuilder    stressBuilder
+	Checker          Checker
+	consistencyCheck bool
```

Comment:
This should be check bool. If this is false we should do no checking. Ideally we can get rid of this flag by having empty checker for no checking case.